src/pages/workflows/workflow/WorkflowSummary.ts

```diff
@@ -57,35 +47,53 @@ export const BaseWorkflowSummary = () => {
         : div({ style: { fontStyle: 'italic' } }, ['No documentation provided']),
     ]),
     div({ style: Style.dashboard.rightBox }, [
-      h2({ style: Style.dashboard.header }, ['Snapshot information']),
-      div({ style: { display: 'flex', flexWrap: 'wrap', margin: -4 } }, [
-        h(InfoTile, { title: 'Creation date' }, [new Date(createDate).toLocaleDateString()]),
-        h(InfoTile, { title: 'Public' }, [_.startCase(isPublic as unknown as string)]),
-      ]),
-      h2({ style: Style.dashboard.header }, ['Owners']),
-      _.map((email) => {
-        return div({ key: email, style: { overflow: 'hidden', textOverflow: 'ellipsis' } }, [
-          h(Link, { href: `mailto:${email}` }, [email]),
-        ]);
-      }, managers),
-      div({ style: { margin: '1.5rem 0 1rem 0', borderBottom: `1px solid ${colors.dark(0.55)}` } }),
-      h2({ style: { fontSize: '1rem', fontWeight: 500, marginBottom: '0.5rem' } }, ['Import URL']),
-      div({ style: { display: 'flex' } }, [
-        div({ style: Style.noWrapEllipsis }, [importUrl]),
-        h(
-          Link,
-          {
-            style: { margin: '0 0.5rem', flexShrink: 0 },
-            tooltip: 'Copy import URL',
-            onClick: withErrorReporting('Error copying to clipboard')(async () => {
-              await clipboard.writeText(importUrl);
-              setImportUrlCopied(true);
-              setTimeout(() => setImportUrlCopied, 1500);
-            }),
-          },
-          [icon(importUrlCopied ? 'check' : 'copy-to-clipboard')]
+      h(
+        RightBoxSection,
+        {
+          title: 'Snapshot Information',
+          defaultPanelOpen: true,
+          persistenceId: `${persistenceId}/snapshotInfoPanelOpen`,
+        },
+        [
+          h(InfoRow, { title: 'Creation Date' }, [new Date(createDate).toLocaleDateString()]),
+          h(InfoRow, { title: 'Publicly Readable' }, [
+            span({ style: { display: 'flex', alignItems: 'center' } }, [
+              icon('users', { size: 20 }),
```

Comment:
small nit: What do you think about displaying this users icon only when the snapshot is public? I think it makes sense to show multiple users icon when snapshot is public but not so much when it is private 🤔

Reply:
This might be overkill, but what about showing this icon: https://fontawesome.com/icons/users?f=classic&s=thin. Or even this icon: https://fontawesome.com/icons/users?f=duotone&s=solid

Reply:
Personally, from above 2 options, I like this one better [https://fontawesome.com/icons/users?f=duotone&s=solid](https://fontawesome.com/icons/users?f=duotone&s=solid) as it greys out other "users". But I am also okay with:
- not showing any icon for private snapshot or
- showing a lock icon instead of users icon

No strong opinions either way so I am good with whatever you think would be the best UX here.